front/public/locales/fr/translation.json

```
@@ -429,56 +429,56 @@
         "help": {
           "no-move": "Les aiguillages ne peuvent pas être déplacés, et sont automatiquement placés sur leur premier port.",
-          "select-track": "Sélectionnez une ligne sur la carte"
+          "select-track": "Sélectionnez une voie sur la carte"
         },
         "label": "Outil \"Aiguillage\"",
-        "no-track-picked-yet": "Aucune ligne n'a encore été sélectionnée",
+        "no-track-picked-yet": "Aucune voie n'a encore été sélectionnée",
         "switch-type": "Type d'aiguillage",
-        "untitled-track": "Ligne sans nom"
+        "untitled-track": "Voie sans nom"
       },
       "track-edition": {
         "actions": {
           "add-at-end": "Ajouter les nouveaux points à la fin",
           "add-at-start": "Ajouter les nouveaux points au début",
           "cancel-line": "Retirer le tracé sélectionné",
-          "confirm-delete-line": "Êtes-vous sûr(e) de vouloir supprimer la ligne ?",
-          "delete-line": "Supprimer la ligne",
+          "confirm-delete-line": "Êtes-vous sûr(e) de vouloir supprimer la voie ?",
+          "delete-line": "Supprimer la voie",
           "mode-add-point": "Ajouter un point",
           "mode-delete-point": "Supprimer des points",
           "mode-move-point": "Déplacer les points",
           "reset-line": "Réinitialiser les données",
-          "save-line": "Sauvegarder la ligne",
+          "save-line": "Sauvegarder la voie",
           "toggle-anchoring": "Activer / désactiver l'ancrage automatique"
         },
         "help": {
-          "add-anchor-point": "Cliquez pour ajouter un point au bout de la section de ligne. Cliquez sur la ligne pour ajouter un point intermédiaire.",
-          "add-point": "Cliquez pour ajouter un point au bout de la section de ligne.",
-          "delete-point": "Cliquez sur un point pour le supprimer de la ligne.",
+          "add-anchor-point": "Cliquez pour ajouter un point au bout de la section de voie. Cliquez sur la voie pour ajouter un point intermédiaire.",
+          "add-point": "Cliquez pour ajouter un point au bout de la section de voie.",
+          "delete-point": "Cliquez sur un point pour le supprimer de la section de voie.",
           "move-point": "Cliquez une première fois sur un point pour commencer à le déplacer.",
           "move-point-end": "Cliquez pour valider le déplacement du point."
         },
-        "attached-electrifications": "Électrifications",
+        "attached-electrifications": "Électrification",
         "attached-speed-sections": "Vitesses limites",
-        "default-electrifications-error": "Une erreur est survenue lors du chargement des {{type}} liées à cette section de ligne.",
-        "default-speed-sections-error": "Une erreur est survenue lors du chargement des vitesses limites liées à cette section de ligne.",
-        "label": "Outil \"Ligne\"",
+        "default-electrifications-error": "Une erreur est survenue lors du chargement des {{type}} liées à cette section de voie.",
```

Do we know if all the `type`will be feminine ?

```suggestion
        "default-electrifications-error": "Une erreur est survenue lors du chargement des {{type}} liés à cette section de voie.",
```

---

I don't know, I guess it's feminine if the type is always 'électrification'

---

Actually it's weird, we use `{{type}}` but when we use this trad key, we never pass anything to `type` like with `count`. Not sure this trad will work as expected.  Or is `type` a special key in i18n that I don't know about ?